accumulate_test.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"github.com/cloudscale-ch/cloudscale-go-sdk/v2"
+	"testing"
+	"time"
+)
+
+func Test(t *testing.T) {
```

Comment:
By convention, tests are named after the function
```suggestion
func TestAccumulateBucketMetricsForObjectsUser(t *testing.T) {
```